coz-profilerConfig.cmake.in

```diff
@@ -1,11 +1,8 @@
 
 
-set(COZDIR destdir)
-
-set(COZ_BIN ${COZDIR}/bin/coz)
-set(COZ_INCLUDE_DIR ${COZDIR}/include)
-get_filename_component(LIBRARY_DIR ${CMAKE_CURRENT_LIST_FILE} DIRECTORY)
-set(COZ_LIBRARY ${LIBRARY_DIR}/libcoz.so)
+set(COZ_BIN @CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_BINDIR@/coz)
```

Comment:
I have fixed the coz-profilerConfig.cmake to be relocatable.
In respect of the coz-profilerConfig.cmake file, I am going to keep the original code. I agree, the way the file is done is not a common practice and there is a better way to do this. This can be made in a separate merge request.